### ISSUE TITLE
react refresh not work with injected require by other library

### ISSUE THREAD

**Issue by levinqdl** [pmmmwh/react-refresh-webpack-plugin#287]
I want to use linaria, which extracts css tag template into a css file and injects a require of the generated css at end of file. But react refresh can't work with it.

I've created a reproduce repo: https://github.com/levinqdl/refresh-reproduce.git

App.js transpiles to:
``` javascript
__webpack_require__.$Refresh$.runtime = __webpack_require__(/*! ./node_modules/react-refresh/runtime.js */ "./node_modules/react-refresh/runtime.js");
__webpack_require__.$Refresh$.setup(module.id);




var App = function App() {
  return /*#__PURE__*/react__WEBPACK_IMPORTED_MODULE_0__.createElement("div", {
    className: "dm0as6w"
  }, "App1");
};

_c = App;
/* harmony default export */ const __WEBPACK_DEFAULT_EXPORT__ = (App);

__webpack_require__(/*! ../.linaria-cache/src/App.linaria.css */ "./.linaria-cache/src/App.linaria.css");

var _c;

__webpack_require__.$Refresh$.register(_c, "App");

const currentExports = __react_refresh_utils__.getModuleExports(module.id);
__react_refresh_utils__.registerExportsForReactRefresh(currentExports, module.id);
```

This issue should be related to #140 , https://github.com/pmmmwh/react-refresh-webpack-plugin/blob/3214268472a5d7f1bd9f27949f9bcb4a0b6d1dd0/lib/runtime/RefreshRuntimeModule.js#L25

__webpack_require__.$Refresh$.register is reseted by __webpack_require__ of css file after __webpack_require__.$Refresh$.setup invoked.

**Comment by pmmmwh** [comment]
Thanks for the report, I will take a look and see if this is actually a regression

**Comment by pmmmwh** [comment]
Fixed in `0.5.0-beta.1`.
